modules/sdk-core/src/bitgo/utils/tss/baseTSSUtils.ts

```diff
@@ -163,6 +165,40 @@ export default class BaseTssUtils<KeyShare> extends MpcUtils implements ITssUtil
     return unsignedTx;
   }
 
+  /**
+   * Create a tx request from params for message signing
+   *
+   * @param params
+   * @param apiVersion
+   * @param preview
+   */
+  async createTxRequestWithIntentForMessageSigning(
+    params: IntentOptionsBase,
+    apiVersion: TxRequestVersion = 'full',
+    preview?: boolean
+  ): Promise<TxRequest> {
+    const intentOptions: PopulatedIntentBase = {
+      intentType: params.intentType,
+      sequenceId: params.sequenceId,
+      comment: params.comment,
+      memo: params.memo?.value,
+      isTss: params.isTss,
```

Comment:
should this always be true?

Reply:
I set it to true when I call this function from wallet. I want to leave the option for this to happen for non-tss wallet in the future.